fjord/feedback/tests/test_api.py

```diff
@@ -1,3 +1,5 @@
+# This Python file uses the following encoding: utf-8
+
```

Comment:
I used one of the ways documented in pep 263 - "Without interpreter line, using plain text" since the style you have quoted was mentioned under "With interpreter binary and using Emacs style file encoding" and I was not sure whether to use it without the interpreter binary shebang line. I will fix it now.